astropy/io/fits/connect.py

```diff
@@ -312,8 +312,10 @@ def _encode_mixins(tbl):
     # meta data which is extracted with meta.get_yaml_from_table.  This ignores
     # Time-subclass columns and leave them in the table so that the downstream
     # FITS Time handling does the right thing.
-    encode_tbl = serialize._represent_mixins_as_columns(
-        tbl, exclude_classes=(Time,))
+
+    with serialize_context_as('fits'):
```

Comment:
Note: this doesn't change anything since the default is the same as `fits`, but it is technically correct and in theory other mixins might need it.